Nasal/light-manager.nas

```diff
@@ -0,0 +1,303 @@
+# provides relative vectors from eye-point to aircraft lights
+# in east/north/up coordinates the renderer uses
+
+var light_manager = {
+    run: 0,
+
+    lat_to_m: 110952.0,
+    lon_to_m: 0.0,
+
+    light1_xpos: 0.0,
```

Comment:
In case we're talking about different things. What I see is this (in pseudocode):

```javascript

class X {
    foo: 0;

    function init () {
        me.foo = "bar";
    }
}
```

Why not write:

```javascript

class X {
    function init () {
        me.foo = "bar";
    }
}
```

?

> It would then make it difficult at best to quickly see what a light_manager object consists of and what are we really gaining?

If you initialize a field in just one place. Doesn't that make it easier?